### ISSUE TITLE
[BUG] - cannot use aria-checked on Button

### ISSUE THREAD

**Issue by keul** [nextui-org/nextui#1620]
### NextUI Version

2.1.12

### Describe the bug

I've a design which requires buttons to work like checkboxes, but I've an open eye on accessibility.

The idea is to use WAI-ARIA attributes, but it seems that nextui is not applying all of the attributes:

```jsx
      <Button role="checkbox" aria-checked="false">
        next button
      </Button>
```

The `role` attribute is passed through, while `aria-checked` isn't.

### Your Example Website or App

https://codesandbox.io/s/nextui-swallowing-aria-hjgjgs

### Steps to Reproduce the Bug or Issue

1. go to https://codesandbox.io/s/nextui-swallowing-aria-hjgjgs
2. inspect the generated markup

### Expected behavior

Any valid ARIA attribute should be passed through

### Operating System Version

MacOS

### Browser

Firefox
